plugins/vpp/l3plugin/vppcalls/vpp1908/route_dump.go

```diff
@@ -148,21 +125,9 @@ func (h *RouteHandler) dumpRouteIPDetails(tableID uint32, tableName []byte, addr
 
 			// Route metadata
 			meta := &vppcalls.RouteMeta{
-				TableName:         bytesToString(tableName),
-				OutgoingIfIdx:     ifIdx,
-				NextHopID:         path.NextHopID,
-				IsIPv6:            ipv6,
-				RpfID:             path.RpfID,
-				Afi:               path.Afi,
-				IsLocal:           uintToBool(path.IsLocal),
```

Comment:
It seems like we are no longer setting all of this information during dumps and still keeping the fields. Seems like a bug.

The new API is defining various new fields for `FibPath` which actually contain all of this information.